keras/objectives.py

```diff
@@ -63,6 +63,36 @@ def cosine_proximity(y_true, y_pred):
     y_pred = K.l2_normalize(y_pred, axis=-1)
     return -K.mean(y_true * y_pred, axis=-1)
 
+# The actual pinball/ tilted loss function for quantile regression
+def tilted_loss(q, y, f):
```

Comment:
Will get onto fixing syntax. But not sure if this loss function is implemented in sklearn, nor do I know where to look (sorry). This is just equation 1 of http://www-personal.acfr.usyd.edu.au/f.ramos/Fabio_Ramos_Homepage/Publications_files/GPquantile.pdf (it is my paper but its a fairly standard loss for quantile regression).

To some extent it will give you an uncertainty estimate of the predicted function (by taking say the 5th and 95th percentile).